src/web/server.py

```diff
@@ -202,6 +202,27 @@ def get(self, user, script_name):
         self.write(json.dumps(config))
 
 
+    @requires_admin_rights
+    @inject_user
+    def delete(self, user, script_name):
+        try:
+            self.application.config_service.delete_config(user, script_name)
+        except ConfigNotAllowedException:
+            LOGGER.warning(
+                f'Admin access to the script "{script_name}" is denied for {user.get_audit_name()}'
+            )
+            respond_error(self, 403, 'Access to the script is denied')
+            return
+        except NotFoundException as e:
+            LOGGER.warning(f'Failed to delete script {script_name}', exc_info=True)
+            respond_error(self, 404, str(e))
+            return
+        except InvalidAccessException as e:
+            raise tornado.web.HTTPError(403, reason=str(e)) from e
+
+        self.set_status(204)
```

Comment:
I would remove this line. Currently, script server always responds with 200 in all the endpoints.
And to be honest, I don't see any benefits of success http codes.

Reply:
Yeah, I'm fine with that. Less code is good code.